<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-06-01T01:30:24 IST" name="Default suite" finished-at="2021-06-01T01:30:45 IST" duration-ms="21909">
    <groups>
    </groups>
    <test started-at="2021-06-01T01:30:24 IST" name="Default test" finished-at="2021-06-01T01:30:45 IST" duration-ms="21909">
      <class name="test_scripts.Valid_Login">
        <test-method is-config="true" signature="openAppln()[pri:0, instance:test_scripts.Valid_Login@40cb8df7]" started-at="2021-06-01T01:30:24 IST" name="openAppln" finished-at="2021-06-01T01:30:41 IST" duration-ms="16925" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openAppln -->
        <test-method signature="loginApp()[pri:0, instance:test_scripts.Valid_Login@40cb8df7]" started-at="2021-06-01T01:30:41 IST" name="loginApp" finished-at="2021-06-01T01:30:41 IST" duration-ms="65" status="FAIL">
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[org/apache/commons/compress/archivers/zip/ZipFile]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: org/apache/commons/compress/archivers/zip/ZipFile
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:312)
at org.apache.poi.xssf.usermodel.XSSFWorkbookFactory.create(XSSFWorkbookFactory.java:97)
at org.apache.poi.xssf.usermodel.XSSFWorkbookFactory.create(XSSFWorkbookFactory.java:36)
at org.apache.poi.ss.usermodel.WorkbookFactory.lambda$create$2(WorkbookFactory.java:199)
at org.apache.poi.ss.usermodel.WorkbookFactory.wp(WorkbookFactory.java:295)
at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:199)
at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:163)
at generic.Generic_read_excel.getData(Generic_read_excel.java:16)
at test_scripts.Valid_Login.loginApp(Valid_Login.java:14)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.compress.archivers.zip.ZipFile
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:606)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:168)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
... 37 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp -->
        <test-method is-config="true" signature="closeAppln(org.testng.ITestResult)[pri:0, instance:test_scripts.Valid_Login@40cb8df7]" started-at="2021-06-01T01:30:41 IST" name="closeAppln" finished-at="2021-06-01T01:30:45 IST" duration-ms="4806" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginApp status=FAILURE method=Valid_Login.loginApp()[pri:0, instance:test_scripts.Valid_Login@40cb8df7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeAppln -->
      </class> <!-- test_scripts.Valid_Login -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
